plugins/premium/comments/comments_end-user_docs.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Can we rename this file please? It doesn't match the title which is confusing.

Reply:
Fixed in [`658e875` (#2007)](https://github.com/tinymce/tinymce-docs/pull/2007/commits/658e8758c071918697232fb55fb7df05f41c6960)